### DISCUSSION TITLE
I cant run wakatime plugin on cpeditor ,the data wasn't transported to the wakatime website,  but in vs2022 it is functional

### DISCUSSION THREAD

**Discussion by Aspirin0000** [cpeditor/cpeditor#1246] [Q&A] [answered]
Can someone tell me which version of wakatime should be used for windows(x64) (intel) ?     Need I use a net proxy in cpeditor?
If you are successful ,please tell me how did you do. Thanks.

**Comment by ouuan** [comment]
Use `wakatime-cli-windows-amd64.zip` at https://github.com/wakatime/wakatime-cli/releases/latest

**Reply by Aspirin0000** [reply]
Thanks .But the problem still exists.

**Comment by ouuan** [comment]
What's your WakaTime setting in CP Editor? Is the path to wakatime-cli in the PATH environment variable?

**Reply by ouuan** [reply]
OK. I'm not sure what the problem is, but that red line on the API key is not effective, and I recommend deleting this reply.

**Reply by Aspirin0000** [reply]
ok thanks

**Reply by Aspirin0000** [reply]
You are a veritable student majoring in information security

**Reply by ouuan** [reply]
I just realized that it could still be visible in notification emails, and there are many people watching this repository. If you are concerned, I recommend generating a new API key on WakaTime.

**Reply by Aspirin0000** [reply]
Thanks , I'll give it a try

**Reply by Aspirin0000** [reply]
So is your wakatime functional?

**Reply by ouuan** [reply]
> So is your wakatime functional?

Yes. I'm using Arch Linux and self-hosted Wakapi though.

**Reply by Aspirin0000** [reply]
That's cool. I will learn next time

**Comment by Aspirin0000** [comment, marked as answer]
OK guys,I know the problem. I always use a unnamed cpp file in cpeditor.  When I use a saved cpp file wakatime will automatically track.
